cpp/cmake/thirdparty/get_arrow.cmake

```
@@ -221,7 +222,7 @@ function(find_and_configure_arrow VERSION BUILD_STATIC ENABLE_S3 ENABLE_ORC ENAB
         # Set this to enable `find_package(Parquet)`
         set(Parquet_DIR "${Arrow_DIR}")
       endif()
-      # Set this to enable `find_package(ArrowDataset)`
+      # Set this to enable `find_package(ArrowDataset)` Will call find_package(ArrowAcero) for us
```

This would make more sense as two sentences. I assume it was written on two lines and the comment was wrapped by a formatter?
```suggestion
      # Set this to enable `find_package(ArrowDataset)`. This will call find_package(ArrowAcero) for us
```

---

Yes the formatter was very helpful here :)